contracts/xsgd_contract.scilla

```diff
@@ -370,11 +375,10 @@ transition lawEnforcementWipingBurn(address : ByStr20, initiator : ByStr20)
       | Some b =>
         balances[address] := zero;
         currentSupply <- totalSupply;
-        new_supply = builtin sub currentSupply bal;
+        new_supply = builtin sub currentSupply b;
         totalSupply := new_supply;
         e = { _eventname : "lawEnforcementWipingBurnt"; blacklister : initiator; address : address; amount : bal};
         event e
-      end
```

Comment:
We could revert this so that QuantStamp has an easier time as it is contained, this is fixed in https://github.com/Xfers/XSGD-scilla/pull/14